src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<system-page>
 		<div class="page-content" style="max-width: 20rem;">
-			<div v-if="passwordless">
```

Comment:
It looks like `passwordless` isn't being used in the component anymore?

Reply:
That's true now for the time being. I'd still like to keep this var initialized so we know it's present and available.